ArduPlane/ArduPlane.cpp

```diff
@@ -84,6 +84,9 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(avoidance_adsb_update,  10,    100),
     SCHED_TASK(button_update,           5,    100),
     SCHED_TASK(stats_update,            1,    100),
+    #if GRIPPER_ENABLED == ENABLED
+        SCHED_TASK_CLASS(AP_Gripper, &plane.g2.gripper, update, 10, 75),
```

Comment:
Consistency would say to not tab in here any further, (although I get why you did it)